resources/views/tasks.blade.php

```diff
@@ -0,0 +1,64 @@
+@extends('layouts.app')
+
+@section('content')
+    <!-- Create Task Form... -->
+    <div class="panel-body">
+        <!-- Display Validation Errors -->
+        @include('common.errors')
+
+        <!-- New Task Form -->
+        {{ Form::open(['route' => 'task.store']) }}
+            <div class="form-group">
+                {!! Form::label('name',  trans('messages.task'), ['class' => 'col-sm-3 control-label'] ) !!}
+                <div class="col-sm-6">
+                    {{ Form::text('name', null, ['class'=>'form-control']) }}
+                </div>
+            </div>
+            <div class="form-group">
+                <div class="col-sm-offset-3 col-sm-6">
+                    {!! Form::button(trans('messages.addTask'), ['type' => 'submit', 'class'=>'btn btn-default fa fa-plus']) !!}
```

Comment:
Em phân biệt cho chị `{{ }}` và `{!! !!}`

Reply:
Dạ hiện tại thì em chưa rõ.

Reply:
thì search đi e?

Reply:
{{ }} cái ni thì trả về dạng HTML, còn {!! !!} thì in ra luôn. Không biết có đúng không :D